cmd/aws-event-sources-controller/cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+steps:
+
+- name: gcr.io/kaniko-project/executor:latest
+  args:
+  - --dockerfile=cmd/${_CMD}/Dockerfile
+  - --destination=gcr.io/$PROJECT_ID/${_CMD}:${COMMIT_SHA}
+  - --destination=gcr.io/$PROJECT_ID/${_CMD}:${_KANIKO_IMAGE_TAG}
+  - --cache-repo=gcr.io/$PROJECT_ID/${_CMD}/cache
+  - --cache=${_KANIKO_USE_BUILD_CACHE}
+  - --no-push=${_KANIKO_NO_PUSH}
+  - ${_KANIKO_EXTRA_ARGS}
+  waitFor: ['-']
+
+timeout: 600s
+
+substitutions:
+  _CMD: aws-event-sources-controller
+  _KANIKO_IMAGE_TAG: latest
+  _KANIKO_NO_PUSH: 'false'
+  _KANIKO_USE_BUILD_CACHE: 'true'
+  _KANIKO_EXTRA_ARGS: ''
+
+options:
+  substitution_option: ALLOW_LOOSE
+  machineType: N1_HIGHCPU_8
```

Comment:
@sameersbn this doesn't seem to be set in every project, do you remember if there was a reason for using 8-CPU machines here?

Reply:
I used that early on to be able to speed up the build. That was when the builds were serial and would time out after 10 minutes.

Reply:
OK, I'm removing it then. Builds shouldn't take _that_ long, even on a single CPU.

Reply:
@antoineco Since we've split it in separate cloudbuild.yaml files we should not need this. However I would set a higher build timeout to avoid failures due to slow network operations of degraded CPU performance from the build machines.